cobbler/actions/buildiso/standalone_buildiso.py

```diff
@@ -0,0 +1,267 @@
+        cfglines.append("LABEL %s" % descendant.name)
+        if menu_indent:
+            cfglines.append("  MENU INDENT %d" % menu_indent)
+        cfglines.append("  MENU LABEL %s" % descendant.name)
+        cfglines.append("  kernel %s" % os.path.basename(distro.kernel))
+
+        cfglines.append(_generate_append_line_standalone(data, distro, descendant))
+
+        autoinstall_data = self._generate_autoinstall_data(descendant, distro, airgapped, data, repo_names_to_copy)
+        autoinstall_name = os.path.join(self.isolinuxdir, "%s.cfg" % descendant.name)
+        with open(autoinstall_name, "w+") as autoinstall_file:
+            autoinstall_file.write(autoinstall_data)
+
+    def generate_standalone_iso(self, distro_name: str, filesource: str, airgapped: bool):
+        """Creates the ``isolinux.cfg`` for a standalone or airgapped ISO image. And copies possible repositories to
+        the image source folder.
+
+        :param distro_name: The name of the Cobbler distribution.
+        :param filesource: The source directory for the ISO. This gets rsynced into isolinuxdir.
+        :param airgapped: Whether the repositories have to be locally available or the internet is reachable.
+        """
+        # Get the distro object for the requested distro and then get all of its descendants (profiles/sub-profiles/
+        # systems) with sort=True for profile/system hierarchy to allow menu indenting
+        distro = self.api.find_distro(name=distro_name)
+        if distro is None:
+            raise ValueError("Distro \"%s\" was not found, aborting generation of ISO-file!" % distro_name)
+
+        filesource = self._validate_standalone_filesource(filesource, distro)
+
+        self.copy_boot_files(distro, self.isolinuxdir)
+
+        self.logger.info("generating an isolinux.cfg")
+
+        cfglines = [self.iso_template]
+
+        repo_names_to_copy: Dict[str, str] = {}
+
+        for descendant in distro.children:
+            descendant = self.api.find_items(what="", name=descendant)
+            # if a list of profiles was given, skip any others and their systems
+            if len(self.profiles) > 0 \
+                    and ((descendant.COLLECTION_TYPE == 'profile' and descendant.name not in self.profiles)
+                         or (descendant.COLLECTION_TYPE == 'system' and descendant.profile not in self.profiles)):
+                continue
+            self._generate_descendant(descendant, cfglines, distro, airgapped, repo_names_to_copy)
+
+        self.logger.info("done writing config")
+        cfglines.append("")
+        cfglines.append("MENU END")
+        with open(os.path.join(self.isolinuxdir, "isolinux.cfg"), "w+") as cfg:
+            cfg.writelines(cfglines)
+
+        self._sync_airgapped_repos(airgapped, repo_names_to_copy)
+
+        # copy distro files last, since they take the most time
+        cmd = "rsync -rlptgu --exclude=boot.cat --exclude=TRANS.TBL --exclude=isolinux/ %s/ %s/../" \
```

Comment:
It's better to split the arguments for `subprocess_call`.

Reply:
Should be fixed with a7f3add